GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj

```diff
@@ -278,13 +304,60 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		0E8625BE155C7EE0E873F36C /* [CP] Check Pods Manifest.lock */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
```

Comment:
these look like not needed to checkin. have you run `pod deintegrate`

Reply:
deintegrate in new commit. Thanks!